lib/provider-metadata.coffee

```diff
@@ -11,6 +11,14 @@ class ProviderMetadata
     if providerBlacklist = @provider.providerblacklist?['autocomplete-plus-fuzzyprovider']
       @disableDefaultProviderSelectors = Selector.create(providerBlacklist)
 
+  matchesEditor: (editor) ->
+    if @provider.textEditorSelectors?
+      atom.views.getView(editor).matches(
+          Array.from(@provider.textEditorSelectors).join(', '))
```

Comment:
I'm wondering about `textEditorSelectors` being a property rather than a method we call on the provider. Thoughts on that?